packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -68,8 +68,8 @@ public class TableColumnText extends TableColumn<TableColumnTextCellRecord, Tabl
 
     public cellRecordFieldNames = ['value'] as const;
 
-    public getDataRecordFieldNames(): string[] {
-        return [fieldName];
+    public dataRecordFieldNames(): readonly string[] {
+        return [fieldName]  as const;
     }
```

Comment:
This would need to match the text column implementation:
```ts
    protected fieldNameChanged( ): void {
        this.dataRecordFieldNames = [this.fieldName] as const;
    }
```